### PR TITLE
feat: genesis migration docs and scripts

### PR DESCRIPTION
**Pull request by daeMOn63**

### CODE REVIEW

File: scripts/staking/collect_staked.js [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+        };
+
+        //console.log(`USER ADDRESS, USER PUBKEY, USER FETCH ADDRESS, TOTAL[afet], PRINCIPAL WHOLE[afet], COMPOUND INTEREST(LIQUID + LOCKED + STAKED)[afet]`);
+        for (const [key, user] of Object.entries(retval.staking.users)) {
+            await user.init(interestRates, estimated_end_block);
+
+            const principal = fetToCanonicalFet(user.principalFET_whole);
+            const compound = fetToCanonicalFet(user.compoundInterestFET_whole);
+
+            console.log(`${key},${user.pubkey},${secp256k1UncompressedPubkeyToFetchAddress(user.pubkey)},${principal.add(compound).toFixed()},${principal.toFixed()},${compound.toFixed()}`);
```
Comment (pbukva) [inline scripts/staking/collect_staked.js:302]: I would suggest 2 things here - well, ultiatelly the place to implement them woul be in the `constructor(...)` method of `User` class:
1. relocate derivation of fetch address (= `secp256k1UncompressedPubkeyToFetchAddress(...)` call) in to `User.contructor(...)` and store the address as data member of `User` class
2. Insie `User.contructor(...)` add verification of derived public key by deriving Ethereum address out of it and **comparing** it against `User.address` data-member - these2 value must be equal. If they are not, something is not right - please `assert` that in the constructor.
This assumes that this check is not done already  in the `getPublicKeyFromTxHash(...)` function.

**Review by pbukva — CHANGES_REQUESTED** [review]
see my in-line comment.

File: scripts/staking/collect_staked.js (hunk 1 above)
Comment (daeMOn63) [reply scripts/staking/collect_staked.js]: Thanks, added, and regenerated the sample export - all pubkeys / address pairs verified properly

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+## Pre migration
+
+1) Share this repository with the mainnet validators
+2) Replace and adjust the last remaining placeholders in the procedure (`TODO_` items)
+
+## On Wednesday September, 15th
+
+1) Pause the bridge contract
+2) Stop the network at around 14:00 UTC (height: 2436700)
```
Comment (ejfitzgerald) [inline README.md:21]: Does this need to be updated now?

Comment (ejfitzgerald) [reply README.md:21]: Ignore my stupid comment

**Pull request merged**